operator/backupcontroller/executor.go

```diff
@@ -25,6 +26,14 @@ type BackupExecutor struct {
 	backup *k8upv1.Backup
 }
 
+// BackupPod contains all information nessecary to execute the backupcommands.
```

Comment:
You can remove this now. The return type of the listPods is now `kubernetes.BackupPod[]`.